presto/src/main/java/org/apache/zeppelin/presto/PrestoInterpreter.java

```diff
@@ -0,0 +1,203 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.zeppelin.presto;
+
+import java.sql.*;
+import java.util.List;
+import java.util.Properties;
+
+import org.apache.zeppelin.interpreter.*;
+import org.apache.commons.lang.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import org.apache.zeppelin.interpreter.InterpreterResult.Code;
+import org.apache.zeppelin.scheduler.Scheduler;
+import org.apache.zeppelin.scheduler.SchedulerFactory;
+
+/**
+ * Presto interpreter for Zeppelin.
+ */
+public class PrestoInterpreter extends Interpreter {
+  Logger logger = LoggerFactory.getLogger(PrestoInterpreter.class);
+  int commandTimeOut = 600000;
+
+  static final String PRESTOSERVER_URL = "presto.url";
+  static final String PRESTOSERVER_USER = "presto.user";
+  static final String PRESTOSERVER_PASSWORD = "presto.password";
+
+  static {
+    Interpreter.register(
+      "presto",
+      "presto",
+      PrestoInterpreter.class.getName(),
+      new InterpreterPropertyBuilder()
+        .add(PRESTOSERVER_URL, "jdbc:presto://localhost:8089", "The URL for Presto")
```

Comment:
Instead of hard-coding default, can we first try to get default through environment variables and system properties.
For eg. please see SparkInterpreter.getSystemDefault()
This would be useful because admins can set a defaults and users can just start using presto and not worry about updating properties on the interpreter page.